web/src/app/admin/connectors/[connector]/Sidebar.tsx

```diff
@@ -8,12 +8,41 @@ import { credentialTemplates } from "@/lib/connectors/credentials";
 import Link from "next/link";
 import { useUser } from "@/components/user/UserProvider";
 import { useContext } from "react";
+import { UserRole } from "@/lib/types";
+
+function BackButton({
+  isAdmin,
+  isCurator,
+}: {
+  isAdmin: boolean;
+  isCurator: boolean;
+}) {
+  const buttonText = isAdmin ? "Admin Page" : "Curator Page";
+
+  if (!isAdmin && !isCurator) {
+    console.error(
+      "User is neither admin nor curator, defaulting to curator view"
+    );
+  }
+
+  return (
+    <div className="mx-3 mt-6 gap-y-1 flex-col flex gap-x-1.5 items-center items-center">
```

Comment:
nit: seems like it was already in the code, but duplicate items-center and extraneous gap-y-1

Reply:
Done!